server/routes/admin.js

```diff
@@ -35,9 +35,30 @@ router.post('/categories/add', function (req, res, next) {
     res.redirect("/");
     res.end()
   }
-  categoryClient.addCategory(query, callback);
+  categoryClint.categoryClint(query, callback);
 });
 
+/*===========================
+        Add article
+===========================*/
+router.get('/articles/add', (req, res, next) => {
+  const callback = (error, category) => {
+    res.render('add-articles', {
+      category
```

Comment:
call the variable `categories` - plural, as it is an array of mroe than one category